vnext/Microsoft.ReactNative.Cxx/CppWinRTIncludes.h

```diff
@@ -23,15 +23,15 @@ using Colors = winrt::Windows::UI::Colors;
 } // namespace winrt
 #else
 
-#include <winrt/Microsoft.System.h>
+#include <winrt/Microsoft.UI.Dispatching.h>
 #include <winrt/Microsoft.UI.Xaml.h>
 
 #define XAML_CPPWINRT_NAMESPACE winrt::Microsoft::UI::Xaml
 namespace xaml = winrt::Microsoft::UI::Xaml;
 namespace comp = winrt::Microsoft::UI::Composition;
 namespace ui = winrt::Microsoft::UI;
 namespace winrt {
-namespace system = winrt::Microsoft::System;
+namespace system = winrt::Microsoft::UI::Dispatching;
```

Comment:
We should rename this namespace if we are changing what it points to

Reply:
I think it needs to match the WUX namespace above, not sure if there's a re-name that would make sense for both. @asklar, what do you think?

Reply:
"dispatching" is the more specific of the two, and aligns our code to the future API. Choosing the intersection makes usage a bit clearer imo, since you may not always have everything under "system".

Reply:
Updated both (and their references) to dispatching :)